webportal/src/app/market_list/components/item_card.jsx

```diff
@@ -4,26 +4,30 @@ import PropTypes from 'prop-types';
 import React, { useContext } from 'react';
 import { Text, Stack, getTheme } from 'office-ui-fabric-react';
 import styled from 'styled-components';
-import Card from 'App/components/card';
 import Context from 'App/context';
 import { ReactComponent as DataIcon } from 'App/assets/data.svg';
 import { ReactComponent as JobIcon } from 'App/assets/job.svg';
 import { TYPE_ENUM } from 'App/utils/constants';
 
 const { palette } = getTheme();
 
-const HoverCard = styled(Card)`
+const HoverCard = styled.div`
+  background: ${palette.neutralLighterAlt};
+  box-shadow: rgba(0, 0, 0, 0.06) 0px 2px 4px, rgba(0, 0, 0, 0.05) 0px 0.5px 1px;
   cursor: pointer;
-  min-width: 400px;
+  min-width: 300px;
+  &:hover {
+    background: ${palette.neutralLighter};
+  }
 `;
 
 const ItemCard = props => {
   const { item } = props;
   const { history } = useContext(Context);
 
-  const populateCreatedTime = () => {
+  const populateUpdateTime = () => {
```

Comment:
>populateUpdateTime [](http://example.com/codeflow?start=8&length=18)

why do you change this name from create to update, and at line 32, the varable is still createdTime, which may cause some confusions...

Reply:
It used in: https://github.com/microsoft/openpaimarketplace/blob/0f6ada07972b381cfd6e00b069ca165a28c5f321/webportal/src/app/market_list/components/item_card.jsx#L55
which changed from `create` to `update`, so this function is also renamed.

Reply:
yes, I found it out. But what is the purpose of changing that? :smile: Besides, the variant name at line 29 is still createdTime, so should we update it as well?

Reply:
There is a UI logic change that createTime should be changed to updateTime
As your advice, update `createdTime` to `updateTime`